config/routes.rb

```diff
@@ -1,4 +1,46 @@
 Rails.application.routes.draw do
+  get 'enrollments/index'
```

Comment:
It's better to use [resources](https://api.rubyonrails.org/v7.0.4.2/classes/ActionDispatch/Routing/Mapper/Resources.html#method-i-resources) instead of writing for each request

Reply:
thanks @vicmaburrito